internal/neofs/tree.go

```diff
@@ -1022,7 +1025,18 @@ func (c *TreeClient) GetLastPart(ctx context.Context, bktInfo *data.BucketInfo,
 		return 1
 	})
 
-	return parts[len(parts)-1], nil
+	var pi *data.PartInfo
+	for _, part := range parts {
```

Comment:
Can be a lot of parts.

Reply:
Yes, it can. Right now there is no mechanism to reduce their amount. We may have separate issue for this